dali/pipeline/operators/operator.h

```diff
@@ -146,7 +147,7 @@ class DLL_PUBLIC OperatorBase {
   using OperatorBase::spec_;                         \
   using OperatorBase::num_threads_;                  \
   using OperatorBase::batch_size_;                   \
-  using OperatorBase::default_cuda_stream_priority_
+  using OperatorBase::default_cuda_stream_priority_  \
```

Comment:
Is that `\` needed here?

Reply:
No, added it by a mistake here. Will remove.